ee/clickhouse/queries/experiments/funnel_experiment_result.py

```diff
@@ -292,3 +296,27 @@ def calculate_probability_of_winning_for_each(variants: List[Variant]) -> List[P
     total_test_probabilities = sum(probabilities[1:])
 
     return [max(0, 1 - total_test_probabilities), *probabilities[1:]]
+
+
+def validate_event_variants(filtered_results, variants):
+    if not filtered_results or not filtered_results[0]:
+        raise ValidationError("No experiment events have been ingested yet")
+
+    eventsWithOrderZero = []
```

Comment:
There needs to always be only one event with order 0

Reply:
Sorry if this comment wasn't clear, I meant we don't need an array here, this should be just one specific event, keeps the data model clearer

Reply:
Nah I'm confused, we'll have multiple when we breakdown, one for each breakdown value, so this makes sense, is correct 🤦